src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/EventHandlers/OverlappingTrainingDateRequest/OverlappingTrainingDateForActiveApprenticeshipEventHandler.cs

```diff
@@ -65,11 +65,10 @@ private SendEmailToEmployerCommand BuildEmailToEmployerCommand(Apprenticeship ap
                 "OverlappingTrainingDateForActiveApprenticeship",
                 new Dictionary<string, string>
                 {
-                        {"EMPLOYERNAME", apprenticeship.Cohort.AccountLegalEntity.Name},
                         {"ULN",message.Uln},
                         {"APPRENTICENAME", $"{apprenticeship.FirstName} {apprenticeship.LastName}"},
                         {"URL", $"{_commitmentsV2Configuration.EmployerCommitmentsBaseUrl}/{_encodingService.Encode(apprenticeship.Cohort.EmployerAccountId,EncodingType.AccountId)}/apprentices/{_encodingService.Encode(apprenticeship.Id, EncodingType.ApprenticeshipId)}/details"}
-                }
+                }, null, "Name"
```

Comment:
Can we keep consistent casing on the template tokens at least within the context of a single email template?

Reply:
done